### DISCUSSION TITLE
I have a problem with firebase firestore "where" queries I can't use different field query

### DISCUSSION THREAD

**Discussion by serkansentilay** [invertase/react-native-firebase#6789] [Q&A]
![editbak](https://user-images.githubusercontent.com/99118830/207882527-d267e942-1f11-410a-90f1-5701d57171df.jpg)
![editbak1](https://user-images.githubusercontent.com/99118830/207882537-a3b10510-f17b-4317-a93d-b259635f3dcc.png)
![editbak2](https://user-images.githubusercontent.com/99118830/207882540-5b75713f-110d-4823-8e5b-008ff277ba54.png)

I try a lot of option but I couldn't, what should I do? 
I want id filter and age filter and I'll add location filter another time. Can you explain guys?
and I tried too on the .filter field but not working and I tried too added on the .where right side field but same not working? I can't see filter data

**Comment by serkansentilay** [comment]
![Adsız4](https://user-images.githubusercontent.com/99118830/207921733-7be403c4-6d38-4088-a1ef-c6e86626a641.png)
I changed to onsnapshot but still not working

**Reply by mikehardy** [reply]
This seems like a general firestore question and might be better asked on stackoverflow - I do not believe it will be react-native-firebase specific

**Reply by serkansentilay** [reply]
okay I'll do to that
